EventFilter/CTPPSRawToDigi/python/totemRawToDigi_cff.py

```diff
@@ -1,7 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
 from CondFormats.CTPPSReadoutObjects.TotemDAQMappingESSourceXML_cfi import *
-TotemDAQMappingESSourceXML.mappingFileNames.append("CondFormats/CTPPSReadoutObjects/xml/ctpps_210_mapping.xml")
+totemDAQMappingESSourceXML.configuration = cms.VPSet(
+  # before TS2 (2016)
+  cms.PSet(
+    validityRange = cms.EventRange("1:min - 280385:max"),
+    mappingFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/ctpps_mapping_to_fill_5288.xml"),
+    maskFileNames = cms.vstring()
+  ),
+  # during TS2 (2016)
+  cms.PSet(
+    validityRange = cms.EventRange("280386:min - 281600:max"),
+    mappingFileNames = cms.vstring(),
```

Comment:
In `mappingFileNames` is empty, the corresponding map in EventSetup will be empty. Then, if the CTPPS raw-to-digi is executed and if there are some data from CTPPS FEDs (which should not be the case), these data are ignored:
https://github.com/CTPPS/cmssw/blob/ctpps_conf_daq_mapping/EventFilter/CTPPSRawToDigi/src/RawToDigiConverter.cc#L73
No crash.